src/helpers/utils.tsx

```diff
@@ -675,3 +675,13 @@ export const setDefaultValues = (target: Dictionary, prop: string) => {
   target[prop] = [];
   return target;
 };
+
+export function reverseChronologicalSort(arr: Plan[], sortField: string) {
```

Comment:
- Can we add a docstring here?
- I think this name isn't accurate - it seems this is not just a reverse chronological sort